### ISSUE TITLE
Can't perform move to operation in AllPhotosMode.

### ISSUE THREAD

**Issue by raj10071997** [fossasia/phimpme-android#1693]
**Actual Behaviour**

When trying to move photos one album to another album in AllPhotosMode the app keeps asking the permission to write. But if we try to move it from inside the album it is moved.

**Expected Behaviour**

The image should even be moved from AllPhotosMode.

**Steps to reproduce it**

Open AllPhotosMode and try to move an image.

**Would you like to work on the issue?**

Yes

**Comment by codedsun** [comment]
Copy of #1686

**Comment by codedsun** [comment]
Kindly close this.! We are working on that, try with the issues which are not been taken by anybody there are many of it.

**Comment by raj10071997** [comment]
@codedsun I didn't take your issue, my issue is specific to AllPhotosMode. And your steps to reproduce are specific to LFMainActivity . You should clearly mention that in what places you were seeing those errors. Otherwise many of the errors are just same. Please mention all the places next time where see that error.

**Comment by codedsun** [comment]
@raj10071997 Yup! it's open source we all are for learning something here, np! if you would like to go on further you could solve that issue.There was a glitch as I and @mayank8318 have both solved the same issue. So next time, yes I will take care of this to mention the exact end points.

**Comment by codedsun** [comment]
you can checkout at the issue, what we faced with today.

**Comment by raj10071997** [comment]
@codedsun  I am sorry for being rude. It wasn't my intention to take anyone issue. np.! you can solve the above issue with your issue after all both the issues refer to the same problem of moving files. Go ahead.! If I find something I will be glad to help.
